### DISCUSSION TITLE
QT version use gpu acceleration ?

### DISCUSSION THREAD

**Discussion by Augusto7743** [doublecmd/doublecmd#1549] [General]
Thanks for reading my topic
@alexx2000 .. all good with you ?

An simple question ... between GTK and QT versions what is the version more faster processing file operations and GUI start and use operations ?

Have an nice week for your and double commander users.

**Comment by alexx2000** [comment]
I am OK. 

I don't know. I don't see a significant difference between Qt and GTK versions. I am using Qt 5 version.

**Comment by Augusto7743** [comment]
I have tested both QT and GTK version in LUbuntu ( Ubuntu using LXQT) 20.04.6.
QT version is much more faster for starting the program and others actions as menus and tabs.
Perhaps files operations are faster too.

Thanks for compiling your amazing software using an QT version compatible with Ubuntu previous versions.
I wish an nice week for you and good luck in you path.
